substrate/frame/revive/fixtures/contracts/tracing.rs

```diff
@@ -32,6 +32,18 @@ pub extern "C" fn deploy() {}
 pub extern "C" fn call() {
 	input!(calls_left: u32, callee_addr: &[u8; 20],);
 	if calls_left == 0 {
+		// transfer some value to BOB
+		let _ = api::call(
+			uapi::CallFlags::empty(),
+			&[2u8; 20],
+			u64::MAX, // How much ref_time to devote for the execution. u64::MAX = use all.
+			u64::MAX, /* How much proof_size to devote for the execution. u64::MAX = use
+			           * all. */
```

Comment:
Out of interest why did you format all those like this?

Reply:
nop for some reason rustfmt did, might be running it on a more recent version that the one we have